linux_os/guide/services/avahi/avahi_configuration/avahi_disable_publishing.rule

```diff
@@ -1,23 +1,18 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7
-
 title: 'Disable Avahi Publishing'
 
 description: |-
-    To prevent other mDNS stacks from running, edit <tt>/etc/avahi/avahi-daemon.conf</tt>
-    and ensure the following line appears in the <tt>[server]</tt> section:
-    <pre>disallow-other-stacks=yes</pre>
+    To prevent Avahi from publishing its records, edit <tt>/etc/avahi/avahi-daemon.conf</tt>
+    and ensure the following line appears in the <tt>[publish]</tt> section:
+    <pre>disable-publishing=yes</pre>
```

Comment:
This changes look a little weird to me. Are they different because of OS differences?

Reply:
@redhatrises So what happened in this case, look at this file: [avahi_prevent_port_sharing.rule](https://github.com/OpenSCAP/scap-security-guide/blob/master/linux_os/guide/services/avahi/avahi_configuration/avahi_prevent_port_sharing.rule).

See how it has the same contents as the previous version of this file (the removed stuff)? This file (avahi_disable_publishing.rule) was a copy of that file (avahi_prevent_port_sharing.rule) with nothing changed. Exactly the same diff. So yeah, these changes are valid, and should be valid for both rhel6 and rhel7, but I'd have to go look at the config line (but it'd be hard to imagine it changed much considering the rest is very similar). 

I think this change is valid.